polyfill/lib/ecmascript.mjs

```diff
@@ -640,9 +640,11 @@ export const ES = ObjectAssign({}, ES2020, {
     return ES.GetOption(options, 'overflow', ['constrain', 'balance'], 'constrain');
```

Comment:
I was curious about why `ToTemporalDurationOverflow` didn't get the same treatment as the other methods, but then I noticed that `ToTemporalDurationOverflow` seems to be unused. Should it be removed?

Reply:
Good catch, I opened #2031 for that.